branchtesting/src/main/java/branchexample/App.java

```diff
@@ -4,7 +4,9 @@
 {
     public void aMethod( String[] args )
     {
-    	
+    	if (true) {
+    		// TODO something new
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](http://localhost:9721/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](http://localhost:9721/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=1&issues=AWSk0CPsi8MY4tObJ7vx&open=AWSk0CPsi8MY4tObJ7vx)

[ISSUE_KEY]: AWSk0CPsi8MY4tObJ7vx